modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -583,6 +586,8 @@ public ObjectField updateCustomObjectField(
 
 		ObjectField newObjectField = (ObjectField)oldObjectField.clone();
 
+		_validateEncryptedBusinessType(
```

Comment:
@p-albuquerque could you please ensure if we should validate on update a system object field?

Thanks!